docs/cloud/dev/spryker-cloud-commerce-os/performance-testing-in-staging-enivronments.md

```diff
@@ -14,4 +15,592 @@ If you want to execute a full load test on a production-like dataset and traffic
+
+2. Enter an item name for the new job and select `Freestyle project`. Once you have done that, you can move to the next step with `OK`.
+![screenshot](https://lh3.googleusercontent.com/drive-viewer/AJc5JmR2mN1q7_Du2JZPTw_CFqi9hjYnEqi8XvjXpgidcmcEeIEvUYwiEFX2GAAeLU105pz53guDHqI=w1920-h919)
+
+3. The next step will allow up to input the commands we need to run. While a description is optional, you can choose to set one here. There are also additional settings, such as a display name, which may also be toggled. As you only need the commands to run once, you can move down to the `Build` section to add the three build steps needed.
+
+Click on `Add build step` and select `Execute shell`. A Command field will appear, allowing you to input the following: 
+
+```bash
+APPLICATION_STORE="DE" COMMAND="$PHP_BIN vendor/bin/codecept fixtures -c vendor/spryker-sdk/load-testing" bash /usr/bin/spryker.sh
+```
+
+{% info_block errorBox %}
+
+Once these fixtures have been generated, attempting to rerun them in the future with the default data files found in **/vendor/spryker-sdk/load-testing/tests/_data/** will cause an error. New data should be considered if you wish to regenerate fixtures for whatever reason.
+
+{% endinfo_block %}
+
+You can change the store for which you wish to generate fixtures for (i.e., `AT` or `US`). This command allows Codeception to locate the proper configuration file with the `-c` flag for the load testing tool. Once the fixtures have been generated, the data needs to be republished. We can have Jenkins do that with the same job by adding an additional build step with `Add build step`.
+
+```bash
+APPLICATION_STORE="DE" COMMAND="$PHP_BIN vendor/bin/console publish:trigger-events" bash /usr/bin/spryker.sh
+```
+
+From here, you can either add another build step to toggle the queue worker to run, or you can run the queue worker job already available within Jenkins, i.e. `DE__queue-worker-start`.
+
+```bash
+APPLICATION_STORE="DE" COMMAND="$PHP_BIN vendor/bin/console queue:worker:start -s " bash /usr/bin/spryker.sh
+```
+![screenshot](https://lh3.googleusercontent.com/drive-viewer/AJc5JmTeb8OPIZwA65e57LNg8fq_t7DnQ2T_okTLFBxcljKIXgqXcjWyt9yiCFiPKX50_Nb2LyE__Ao=w1920-h919)
+
+4. Once the build steps have been added, you can `Save` to be taken to the project status page for the newly-created job. As this is a job that you only need to run once and no schedule was set, you can select the `Build Now` option.
+![screenshot](https://lh3.googleusercontent.com/drive-viewer/AJc5JmRTLzDFMolgcaZ_xE-nKMNBEiIDXkSjwEiInkEIJL3ZbMbIY5ygKXqc-7eE_H5N2X-m7ap1l8s=w1920-h919)
+
+5. With the job set to build and run, it will build a new workspace for the tasks and run each build step that you specified. Once the build has successfully completed, you can review the `Console Output` and then remove the project `Delete Project` once you as finished if you no longer need it.
```

Comment:
```suggestion
5. With the job set to build and run, it will build a new workspace for the tasks and run each build step that you specified. Once the build has successfully completed, you can review the `Console Output` and then remove the project with `Delete Project` once you are finished, if you no longer need it.
```